clients/src/main/java/org/apache/kafka/common/requests/FetchResponse.java

```diff
@@ -203,8 +203,7 @@ public static FetchResponseData.PartitionData partitionResponse(int partition, E
      * Returns `partition.records` as `Records` (instead of `BaseRecords`). If `records` is `null`, returns `MemoryRecords.EMPTY`.
      *
      * If this response was deserialized after a fetch, this method should never fail. An example where this would
-     * fail is a down-converted response (e.g. LazyDownConversionRecords) on the broker (before it's serialized and
-     * sent on the wire).
+     * fail is a down-converted response on the broker (before it's serialized and sent on the wire).
```

Comment:
the message conversion has been removed by #18218, so this comment is stale. could you please remove it?